tests/tests/test_legal_reason.py

```diff
@@ -319,3 +320,14 @@ def test_legal_reason_hardcore(self):
         self.assertAnonymizedDataExists(anon_payment, "value")
         assert_not_equal(anon_payment.date, payment.date)
         self.assertAnonymizedDataExists(anon_payment, "date")
+
+    def test_purpose_source_model_class_should_be_set_with_string(self):
+        assert_equal(FacebookPurpose.source_model_class, Customer)
```

Comment:
You are not using string, or?

Reply:
I want to test both casses. So I neet to have tests for both string and class deifinition

Reply:
Yes, but the test method name says it should be set using string, whereas you are setting it by directly passing class reference.

Reply:
FacebookPurpose has set source_model_class with string which is converted to the model. It is right